internal/adapters/runtime/kubernetes/game_room_watcher.go

```diff
@@ -200,5 +235,35 @@ func (k *kubernetes) WatchGameRoomInstances(ctx context.Context, scheduler *enti
 	})
 
 	go podsInformer.Run(watcher.stopChan)
+
+	// Same amount of routines reading from the channel
+	for i := 0; i < 200; i++ {
```

Comment:
In the future, yes, of course
This number does not have a silver bullet, and in my opinion it should not be an environment variable, for example, since it's very hard to infer which number is okay
My approach would be to scale it based on the len of the workqueue. It the workqueue becomes too big, a new worker would be created
This thought, however, are not meant to be implemented now
There are more important features to look for (like self-healing and autoscaling). We should catch up in the near feature